src/Microsoft.IdentityModel.Tokens/Encryption/Cng.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography;
+using Microsoft.IdentityModel.Logging;
+using static Microsoft.IdentityModel.Tokens.Interop.BCrypt;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    internal static class CryptoThrowHelper
```

Comment:
It is OK to keep these related classes in one file.
Please arrange the classes in lexicographic order.

Reply:
Fixed it.